prbt_hardware_support/test/integrationtests/integrationtest_execute_brake_test.cpp

```diff
@@ -73,6 +108,29 @@ TEST(IntegrationtestTriggerBrakeTest, testBrakeTestService)
    **********/
   BrakeTest srv;
   EXPECT_TRUE(brake_test_srv_client_.call(srv));
+
+  /**********
+   * Step 4 *
+   **********/
+  RegCont content_perf, content_res;
```

Comment:
Could you refactor this into a function with 
```suggestion
 EXPECT_TRUE(brakeTestRegisterSetOnServer(server, /*timeout*/, /*retries*/, /*sleep*/)
```
maybe inside testutils or somewhere?

Return  a `::testing::AssertionResult` for better test output on failure